setup.py

```diff
@@ -2,41 +2,10 @@
 import os, re
 from setuptools import setup
 
-readme = open(os.path.join(os.path.dirname(__file__), 'README.rst'), 'r').read()
-
 module_file = open(os.path.join(os.path.dirname(__file__), 'libmodernize', '__init__.py'), 'r').read()
```

Comment:
maybe we should be using flit instead, @takluyver

Reply:
Up to you. I like it, but I'm aware it's not everyone's cup of tea, and I have tried to avoid pushing it in projects I maintain with other people.